.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish crate
```

Comment:
This action is going to fall short as soon as we need to publish any individual crate "out of band" but we can cross that bridge when we get to it.